### DISCUSSION TITLE
Rocket Features

### DISCUSSION THREAD

**Discussion by DhruvDabhi1** [rwf2/Rocket#2686] [Ideas] [answered]
I used Rocket a lot in my coding and Rocket is very good create to use for server side but We use async-graphQL also,

I perform Query and Mutation but Want to use Subscription but as we know Subscription need real time interaction with server So why 
Rocket not able to support Subscription. So my question is when Rocket is fully supporting for Subscription and it already Supported? Then please add few example in example folder. Thank you for giving your Time.

**Comment by Amitpatel7090** [comment]
I agreed, I am also facing  errors in rocket to support Subscription in Graphql server. @DhruvDabhi1

**Comment by ELD** [comment, marked as answer]
From what I understand, async-graphql's subscriptions are a library-level feature they provide for various web frameworks. Prior to Rocket shipping with websocket support, subscriptions weren't possible. Now that websockets are supported in Rocket 0.5, it would be up to async-graphql to add that feature to their Rocket extension.
